src/main/java/com/xebia/headerbuddy/annotations/validators/ValidMethodValidator.java

```diff
@@ -0,0 +1,58 @@
+package com.xebia.headerbuddy.annotations.validators;
+
+import com.xebia.headerbuddy.annotations.ValidMethod;
+import com.xebia.headerbuddy.models.RequestBehaviour;
+import org.eclipse.jgit.util.StringUtils;
+import org.reflections.Reflections;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class ValidMethodValidator implements ConstraintValidator<ValidMethod, String> {
```

Comment:
Method is too long, try breaking it up in several methods with descriptive names which probably remove the need for the comments.